indexer/feature_utils.cpp

```diff
@@ -503,4 +503,41 @@ string FormatElevation(string_view elevation)
   return {};
 }
 
+constexpr char const * kWlan = "wlan";
```

Comment:
Replace with std::string_view?

Reply:
If DebugPrint should return string, then this change doesn't make sense